### PR TITLE
Rename vars

### PR DESCRIPTION
**Pull request by iwi**
# Summary
Just a simple renaming of some variables. In particular `url_cache_list` that has become just `cache` and `url_cache` that has become `element`.

# Review
Agreement with the renaming.
Confirmation that nothing's broken.

### CODE REVIEW

File: linkatos/activities.py [hunk 1]
```diff
@@ -9,71 +9,69 @@ def is_empty(events):
     return ((events is None) or (len(events) == 0))
 
 
-def is_url(url_cache):
-    return url_cache is not None
+def is_url(element):
+    return element is not None
```
Comment (arnau) [inline linkatos/activities.py]: I think, it would be better to put all cache related functions in a dedicated `cache.py` file. `activities.py` feels unrelated to cache internal operations.

File: linkatos/activities.py [hunk 2]
```diff
@@ -9,71 +9,69 @@ def is_empty(events):
...
 
 
 def is_not_from_bot(bot_id, user_id):
     return not bot_id == user_id
 
 
-def is_empty_list(list):
-    return len(list) == 0
+def is_empty_list(ls):
+    return len(ls) == 0
```
Comment (arnau) [inline linkatos/activities.py]: What motivates this change? `ls` is rather unconventional.

Comment (arnau) [inline linkatos/activities.py]: My preferences are, either `list` or `xs` as in "multiple x".

File: linkatos/message.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import re
 
+
 def extract_url(message):
-    """
-    Returns the first url in a message. If there aren't any returns None
-    """
-    answer = re.search("(?:\s|^)<(https?://[\w./?+&+%$!#=\-_]+)>(?:\s|$)", message)
+    # Returns the first url in a message. If there aren't any returns None
```
Comment (arnau) [inline linkatos/message.py]: Why did you change from a docstring to a comment?

File: linkatos/message.py [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+    url_re = "(?:\s|^)<(https?://[\w./?+&+%$!#=\-_]+)>(?:\s|$)"
+    answer = re.search(url_re, message)
```
Comment (arnau) [inline linkatos/message.py]: `result` or `res` is more used than `answer`.

Comment (arnau) [inline linkatos/message.py]: or actually, in this case I would use `captures` as in "group captures" because it is the name of what this variable holds.

File: linkatos/activities.py (hunk 1 above)
Comment (iwi) [inline linkatos/activities.py]: yes, I agree. I'll add an issue for this

Comment (iwi) [inline linkatos/activities.py]: added

File: linkatos/activities.py (hunk 2 above)
Comment (iwi) [inline linkatos/activities.py]: I noticed `list` was a key word and thought it might bring issues later on.
I'll move back to it if you think it's unlikely to be problematic.

File: linkatos/message.py (hunk 3 above)
Comment (iwi) [inline linkatos/message.py]: It was the only docstring in the whole application and I didn't have a reason for the difference.
Is there a common practice regarding this?

File: linkatos/message.py (hunk 4 above)
Comment (iwi) [inline linkatos/message.py]: ok, makes sense

File: linkatos/activities.py (hunk 2 above)
Comment (arnau) [inline linkatos/activities.py]: change it to either `xs` or `collection`. Although unlikely to harm anyone, you are right that shadowing keywords is not a good practice.
